book/embeds.md

```diff
@@ -484,6 +484,21 @@ class BlockLayout:
         self.add_inline_child(node, w, ImageLayout)
 ```
 
+And of course, images also get the same inline layout mode as input elements:
+
+``` {.python replace=%22img%22/%22img%22%2c%20%22iframe%22}
+class BlockLayout:
+    def layout_mode(self):
+        # ...
+        elif self.node.tag in {"input", "img"}:
+            return "inline"
+
+    def should_paint(self):
+        return isinstance(self.node, Text) or \
+            (self.node.tag not in {"input", "button", "img"})
```

Comment:
Can we use a list `["input", "button", "img"]` instead of a set? We don't use sets anywhere else and this way we avoid forcing a tiny bit of Python syntax on the reader. With 3 elements it's plausible that lists are also faster (linear scan on 3 elements vs hash lookup) though in Python everything has such enormous overheads that that kind of logic is never decisive.